src/main/java/scala/compat/java8/wrappers/IteratorPrimitiveDoubleWrapper.java

```diff
@@ -0,0 +1,13 @@
+package scala.compat.java8.wrappers;
```

Comment:
Maybe `scala.collection.java8.primitiveCompat` or somesuch, if we move it into the scala.collection namespace?